sdks/java/core/src/main/java/org/apache/beam/sdk/io/BigQueryIO.java

```diff
@@ -631,13 +631,18 @@ public void populateDisplayData(DisplayData.Builder builder) {
         TableReference table = getTable();
 
         if (table != null) {
-          builder.add(DisplayData.item("table", toTableSpec(table)));
+          builder.add(DisplayData.item("table", toTableSpec(table))
+            .withLabel("Table"));
         }
 
         builder
-            .addIfNotNull(DisplayData.item("query", query))
-            .addIfNotNull(DisplayData.item("flattenResults", flattenResults))
-            .addIfNotDefault(DisplayData.item("validation", validate), true);
+            .addIfNotNull(DisplayData.item("query", query)
+              .withLabel("Query"))
+            .addIfNotNull(DisplayData.item("flattenResults", flattenResults)
+              .withLabel("Flatten Query Results"))
```

Comment:
Interesting. I like, though we don't have a precedent for linking to documentation. Should we make this the first or follow-up with a plan for other transforms?